tests/unit/deploy/test_deployer.py

```diff
@@ -607,3 +609,284 @@ def badview():
 
     with pytest.raises(ValueError):
         validate_routes(sample_app.routes)
+
+
+class TestLambdaInitialDeploymentWithConfigurations(object):
+    @fixture(autouse=True)
```

Comment:
So I pass in the `app_policy` fixture in order to reduce the extra arguments in the test methods by putting it in a `setup()` method. Originally I tried to use the `setup_method` which is the same as `unittest.TestCase.setUp` for pytest, but these methods get called before the fixtures are applied/injected so you get errors like:
```
TypeError: setUp() takes exactly 2 arguments (1 given)
```
when I try to supply the `app_policy` fixture.

Therefore by making it an `autouse=True`, I can pass in any fixture to this `setup` method allowing me to reduce the number of fixtures I have to define in the test methods and have the `setup()` be ran for every test.